src/extension.ts

```diff
@@ -50,9 +50,9 @@ export function activate(context: vscode.ExtensionContext) {
         sshRunner.runPlaybook(playbook ? playbook.fsPath : null);
     }));
 
-    context.subscriptions.push(vscode.commands.registerCommand('vscode-ansible.deployment-templates', (playbook) => {
-        deploymentTemplate.displayDeploymentTemplateMenu();
-    }));
+    // context.subscriptions.push(vscode.commands.registerCommand('vscode-ansible.deployment-templates', (playbook) => {
+    //     deploymentTemplate.displayDeploymentTemplateMenu();
+    // }));
```

Comment:
we could have some secret code in the config. if it's set that would be enabled.

Reply:
that way we could just tell somebody to go to settings and enable preview features.

Reply:
but if user enabled it.....